grizzly/users/messagequeue.py

```diff
@@ -197,14 +213,23 @@ def __init__(self, environment: Environment, *args: Tuple[Any, ...], **kwargs: D
 
         auth_context = self._context.get('auth', {})
         username = auth_context.get('username', None)
+        message_context = self._context.get('message', {})
+        header_type = message_context.get('header_type', None)
+        if header_type:
+            header_type = header_type.lower()
+            if header_type not in ['rfh2', 'none']:
+                raise ValueError(f'{self.__class__.__name__} unsupported value for header_type: "{message_context["header_type"]}", supported ones are "None" and "RFH2"')
```

Comment:
`{message_context["header_type"]}` could be `header_type` in the f-string.

Reply:
check!